server/loggerUtils/logger.ts

```diff
@@ -0,0 +1,53 @@
+import winston from "winston";
+import dotenv from "dotenv";
+dotenv.config();
+
+// Configure logger
+const { combine, errors, timestamp, json, prettyPrint } = winston.format;
+const logLevels = {
+  fatal: 0,
+  error: 1,
+  warn: 2,
+  info: 3,
+  debug: 4,
+  trace: 5,
+};
+
+export const logger = winston.createLogger({
+  levels: logLevels,
+  level: process.env.NODE_ENV !== "dev" ? "info" : "trace",

```

Comment:
Could we change this to be `process.env.NODE_ENV === "production" ? "info" : "trace"`? It would be more consistent with how we are checking this env variable in other parts of the app.